src/libs/actions/Report.js

```diff
@@ -951,12 +891,12 @@ function addAction(reportID, text, file) {
         : htmlForNewComment.replace(/((<br[^>]*>)+)/gi, ' ').replace(/<[^>]*>?/gm, '');
 
     // Update the report in Onyx to have the new sequence number
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {
+    const optimisticReport = {
         maxSequenceNumber: newSequenceNumber,
         lastMessageTimestamp: moment().unix(),
         lastMessageText: ReportUtils.formatReportLastMessageText(textForNewComment),
         lastActorEmail: currentUserEmail,
-    });
+    };
```

Comment:
can we put `optimisticReportActionIDs` in here now rather than generate it and push it in after?

Reply:
We could. I left it as is because I think it makes an easier diff for someone to review or reference later when they check out this PR in the future vs. moving things around for no real benefit.